utilities/transactions/transaction_lock_mgr.cc

```diff
@@ -115,7 +115,7 @@ TransactionLockMgr::TransactionLockMgr(
       mutex_factory_(mutex_factory) {
   assert(txn_db);
   txn_db_impl_ =
-      static_cast_with_check<TransactionDBImpl, TransactionDB>(txn_db);
+      static_cast_with_check<PessimisticTxnDB, TransactionDB>(txn_db);
```

Comment:
I guess multiple of places have `PessimisticTxnDB` instead of `PessimisticTransactionDB`
I think we will hit a compilation error